### ISSUE TITLE
DL3062 false positive on `go install tool`

### ISSUE THREAD

**Issue by medasx** [hadolint/hadolint#1132]
Hey! 

DL3062 returns warning on `go install tool` (available since go1.24). 

> [Release notes - tools](https://tip.golang.org/doc/go1.24#tools) 
>
> The new tool meta-pattern refers to all tools in the current module. This can be used to upgrade them all with go get tool or to install them into your GOBIN directory with go install tool.

**Comment by LiorHalfon** [comment]
I got a false positive too on this line:
```
RUN --mount=type=secret,id=gcloud,target=/key.json GOPROXY=proxy.golang.org go run github.com/GoogleCloudPlatform/artifact-registry-go-tools/cmd/auth@v0.1.0 add-locations --locations=europe-west4 && go run github.com/GoogleCloudPlatform/artifact-registry-go-tools/cmd/auth@v0.1.0 refresh
```

Got:
```
Pin versions in go. Instead of `go install <package>` use `go install <package>@<version>`
```
But as you can see, both commands have "<package>@<version>".
